src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction',
```

Comment:
- [ ] Please update the categories slice for the check status action it should return the string "Under Construction"  and In case of unknown action, it should return the current state as requested in the project requirements.
  - You can consider the below code.
```javascript
 checkStatus: (state, action) => {
      state.categories=
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```

Reply:
Thank you for this suggestion. Very helpful!